### PR TITLE
Add recipe for simple RAG

### PR DESCRIPTION
**Pull request by Westwooo**

### CODE REVIEW

File: docs/commands.adoc [hunk 1]
```diff
@@ -669,6 +669,7 @@ Requires a vector to be used as the source of the search which can be supplied i
 For example imagine we have the following document stored in our cluster:
 
 [options="nowrap"]
+```
```
Comment (ingenthr) [inline docs/commands.adoc:672]: Seems unrelated to this change?

File: docs/recipes/simple_rag.adoc [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+Next we use https://couchbase.sh/docs/#_vector_create_index[vector create-index] to create a vector index over our new vector docs:
+
+```
+👤 Charlie 🏠 remote in ☁️ RagChunks._default._default
+> vector create-index cbsh-docs-index vector 1536
+```
+
+See the previous link of a detailed explanation of this command.
+
+Now that we have craeted and indexed our vector docs we can take a look at the final command we will use: https://couchbase.sh/docs/#_ask[ask].
```
Comment (ingenthr) [inline docs/recipes/simple_rag.adoc:114]: craeted?  :)

Might be good to run a spell checker over these docs in an automated way.  Simon Murray did this for operator and it was actually quite a boon, as it forced us to consistently document technical terms.

**Review by ingenthr — CHANGES_REQUESTED** [review]

File: docs/commands.adoc (hunk 1 above)
Comment (Westwooo) [reply docs/commands.adoc:672]: It's fixing a bug, I think it accidentally got deleted in one of the other recipe PRs

File: docs/recipes/simple_rag.adoc (hunk 2 above)
Comment (Westwooo) [reply docs/recipes/simple_rag.adoc]: Corrected and raised an issue to do this: https://github.com/couchbaselabs/couchbase-shell/issues/442

**Review by ingenthr — APPROVED** [review]
